### PR TITLE
[RFR] fix rhevm template scripts

### PR DESCRIPTION
**Pull request by izapolsk**
template test and upload scripts for rhevm are broken. Fixing that issue

### CODE REVIEW

File: requirements/frozen.py3.txt [hunk 1]
```diff
@@ -218,6 +216,7 @@ pickleshare==0.7.5
 pika==0.12.0
 Pillow==5.3.0
 pluggy==0.6.0
+polarion-docstrings==0.15
```
Comment (mshriver) [inline requirements/frozen.py3.txt]: We can pickup `0.16`

File: requirements/frozen_docs.py2.txt [hunk 2]
```diff
@@ -122,6 +123,7 @@ pyOpenSSL==18.0.0
 pyparsing==2.2.2
 pytesseract==0.2.5
 pytest==3.4.1
+pytest-polarion-collect==0.10
```
Comment (mshriver) [inline requirements/frozen_docs.py2.txt]: We can pickup 0.11

File: requirements/frozen.py3.txt (hunk 1 above)
Comment (izapolsk) [reply requirements/frozen.py3.txt]: freeze_all script picked up this version

File: requirements/frozen_docs.py2.txt (hunk 2 above)
Comment (izapolsk) [reply requirements/frozen_docs.py2.txt]: same here ^^

**Pull request merged**